esti/lakectl_local_test.go

```diff
@@ -60,6 +60,23 @@ func localExtractRelativePathsByPrefix(t *testing.T, prefix string, objs []strin
 	return
 }
 
+func localGetSummary(t *testing.T, tasks local.Tasks) string {
+	result := "Summary:\n\n"
+	t.Helper()
+	switch {
+	case tasks.Downloaded > 0:
+		result += fmt.Sprintf("Downloaded: %d\n", tasks.Downloaded)
+	case tasks.Uploaded > 0:
+		result += fmt.Sprintf("Uploaded: %d\n", tasks.Uploaded)
+	case tasks.Removed > 0:
+		result += fmt.Sprintf("Deleted: %d\n", tasks.Removed)
+	}
```

Comment:
The switch case will break on first match - think you want to append for each case and check for zero first

Reply:
True